examples/simple/lint/L-EXC3-bad.c

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include <stdlib.h>
+#include <stdio.h>
+
+void * func()
+{
+    int * retval = malloc(sizeof(int));
+    *retval = 1;
+    return retval; 
```

Comment:
yeah I'd be inclined to just make the function return a pointer to a statically allocated object, eg:
```
int x;
void *func() { return &x; }
```